chalk-solve/Cargo.toml

```diff
@@ -22,4 +22,4 @@ chalk-ir = { version = "0.10.1-dev", path = "../chalk-ir" }
 chalk-rust-ir = { version = "0.10.1-dev", path = "../chalk-rust-ir" }
 
 [dev-dependencies]
-chalk-integration = { version = "0.10.1-dev", path = "../chalk-integration" }
+chalk-integration = { path = "../chalk-integration" }
```

Comment:
Is that not needed because it's a dev dependency?

Reply:
Yes. And also the tool does not update this version because of that reason.

Reply:
Hmm. So I *was* versioning the non-published crates for consistency. But I guess it's okay if they don't get updated. What is going to be there "permanent version"?

(I don't know if we ever *want* to publish these. They're really just a fancy testing infrastructure)

Reply:
The private crates do get versioned with the action here. It's just that version key in dev-dependencies for crates in workspace don't get updated. The reason behind this is that the version key is meaningless for dev deps if path exists.

Reply:
Oh cool